app/src/main/res/values/themes.xml

```diff
@@ -12,5 +12,13 @@
         <!-- Status bar color. -->
         <item name="android:statusBarColor">?attr/colorPrimaryVariant</item>
         <!-- Customize your theme here. -->
+        <item name="android:textViewStyle">@style/CenturyGothicTextViewStyle</item>
+        <item name="android:fontFamily">@font/century_gothic_bold</item>
+    </style>
+
+    <style name="CenturyGothicTextViewStyle" parent="android:Widget.TextView">
+        <item name="android:fontFamily">@font/century_gothic_bold</item>
+        <item name="android:textSize">16sp</item>
```

Comment:
for textSize, don't do hardcode, wait for dimens to be created, then link the element to this code

Reply:
Yes, I'm writing for a while and going to rebase after MP-195 ready